src/main/java/uk/gov/justice/framework/tools/command/AbstractLogSearcherCommand.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.framework.tools.command;
+
+import uk.gov.justice.log.search.main.output.ConsolePrinter;
+import uk.gov.justice.log.search.main.output.OutputPrinter;
+
+import com.beust.jcommander.Parameter;
+
+abstract class AbstractLogSearcherCommand {
```

Comment:
Followed the pattern from Artemis-manager , but then your right there is only one command right now so removing this class